build.sbt

```diff
@@ -553,6 +556,28 @@ lazy val tests = projectMatrix
   )
   .http4sPlatform(allJvmScalaVersions, jvmDimSettings)
 
+lazy val weaverTests = projectMatrix
+  .in(file("modules/weaver-tests"))
+  .dependsOn(core, http4s % "test->compile")
+  .settings(
+    isCE3 := true,
+    // todo
+    scalacOptions -= "-Xfatal-warnings",
```

Comment:
couldn't you have a global plugin that'd remove this everywhere ?

Reply:
I probably should... :joy:

Reply:
now I remember - last time I tried I came to the conclusion that anything a global plugin does will be overridden by sbt-tpolecat and similar. Maybe it's possible, but I didn't figure it out